lib/shared/cliOptions.js

```diff
@@ -80,4 +81,12 @@ module.exports = {
     desc: chalk.gray(
       'Continue execution of tasks upon failure.'),
   },
+  loglevel: {
+    alias: 'l',
```

Comment:
capital `-L` and capital `-S`?  @tkellen @contra thoughts?  Could change `loglevel` to `verbose` and use capital `-V` if you don't like those...